src/test/java/org/opensearch/sdk/TestExtensionsRunner.java

```diff
@@ -141,7 +139,6 @@ public void testHandleExtensionRestRequest() throws Exception {
 
         String ext = "token_placeholder";
         @SuppressWarnings("unused") // placeholder to test the token when identity features merged
-        Principal userPrincipal = () -> "user1";
```

Comment:
We actually want to leave this here as a placeholder, and it has the suppress warnings to indicate it's not a problem.